app/views/discovered_mailer/_discovered_host.html.erb

```diff
@@ -1,10 +1,18 @@
-<% td_style = "text-align: left; border-collapse: collapse; background-color: #FFFFFF; padding: 4px; border: 1px solid #cccccc;" %>
-<td style="<%= td_style%>">
+<style>
+  .discovery-mailer-td-style {
+    text-align: left;
+    border-collapse: collapse;
+    background-color: #fff;
+    padding: 4px;
+    border: 1px solid #ccc;
+  }
+</style>
+<td class="discovery-mailer-td-style">
   <%= link_to host, discovered_host_path(:id => host, :host => @url.host, :port => @url.port, :only_path => false, :protocol => @url.scheme) %>
 </td>
-<td style="<%= td_style%>"><%= host.try(:hardware_model_name) || 'N/A' %></td>
-<td style="<%= td_style%>"><%= host.ip %></td>
-<td style="<%= td_style%>"><%= discovery_attribute(host, :cpu_count) %></td>
-<td style="<%= td_style%>"><%= number_to_human_size(discovery_attribute(host, :memory, 0) * 1024 * 1024) %></td>
-<td style="<%= td_style%>"><%= discovery_attribute(host, :disk_count) %></td>
-<td style="<%= td_style%>"><%= number_to_human_size(discovery_attribute(host, :disks_size, 0) * 1024 * 1024) %></td>
+<td class="discovery-mailer-td-style"><%= host.try(:hardware_model_name) || 'N/A' %></td>
```

Comment:
can't we find a better css matcher? you can probably take any td inside a discovery table or something like that?

Reply:
I was thinking about [bem](http://getbem.com/introduction/), but didn't want to go all the way, because nothing works in that manner here, so even the name convention is not bem, but hints for it.

I do understand why to choose id instead of class for one element, but if I understand correctly the design, each cell might contain different styling, so I didn't want to break that design.

Is my understanding wrong?